scan.hcl

```diff
@@ -0,0 +1,53 @@
+# Copyright (c) HashiCorp, Inc.
```

Comment:
Run hclfmt on this file please.

Also, this is a super nitpick :grinning:... but is it at all possible to have this somewhere other than the root of the repo? It seems like more and more tools want to pollute the root directory with their little dotfiles and it just turns into a junk drawer. If not possible, not the end of the world

Reply:
hclfmt ✅ 

I think it should be doable in the CI since we have the `SECURITY_SCANNER_CONFIG_FILE` option in the security-scanner. I'll need to double check if this would cause any issues in our scheduled scanners that run twice a month internally, in case it fetches the config from the root of the project. I think @picatz could provide some additional details here?

Reply:
This should normally now be handled by https://github.com/hashicorp/nomad/pull/20465/commits/7c13047e43134afd63ba5904691ab9d4a4b9ee6d